src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplateField/PhrasalTemplateField.styles.ts

```diff
@@ -1,6 +1,13 @@
 import { Box, styled } from '@mui/material';
 
-import { theme, variables } from 'shared/styles';
+import { StyledFlexTopCenter, theme, variables } from 'shared/styles';
+
+export const StyledFlexTopCenterDraggable = styled(StyledFlexTopCenter)`
+  ${({ isDragging }: { isDragging: boolean }) => `
+  box-shadow: ${isDragging ? variables.boxShadow.light3 : 'inherit'};
+  ${isDragging && `background-color: ${variables.palette.surface}`};
+`};
+`;
```

Comment:
Another suggestion to make the dragging style more closely match Figma:
```suggestion
export const StyledFlexTopCenterDraggable = styled(StyledFlexTopCenter)(
  ({ isDragging }: { isDragging: boolean }) => `
  padding: ${theme.spacing(0.8)};
  margin: ${theme.spacing(-0.8)};
  ${
    isDragging
      ? `
      background-color: ${variables.palette.surface};
      border-radius: ${variables.borderRadius.md};
      box-shadow: ${variables.boxShadow.light3};`
      : ''
  }`,
);
```